pkg/hooks/hooks_test.go

```diff
@@ -2,28 +2,21 @@ package hooks
 
 import (
 	"errors"
-	"fmt"
 	"os"
+	"path"
 	"testing"
-	"time"
 
 	"github.com/cozy/cozy-stack/pkg/config/config"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestHooks(t *testing.T) {
-	if testing.Short() {
-		t.Skip("an instance is required for this test: test skipped due to the use of --short flag")
-	}
-
 	config.UseTestFile()
-	config.GetConfig().Hooks = "../../tests/hooks"
-	os.Exit(m.Run())
+	config.GetConfig().Hooks = "./testdata"
```

Comment:
Why did you move the hooks from `tests/hooks` to `pkg/hooks/testdata`?

Reply:
Based on the description of `go help test` the `/testdata` seems to be a more correct/standard place to handle this kind of file. 

```
go help test
...
The go tool will ignore a directory named "testdata", making it available
to hold ancillary data needed by the tests.
...
```

As explained in [the testing package doc](https://pkg.go.dev/testing#hdr-Fuzzing) this is also a good place for the Fuzz testing data feed.  :slightly_smiling_face:

Reply:
By the way, you can remove the hooks from `tests/hooks`, not just copy them